tasks/android.py

```diff
@@ -94,7 +94,7 @@ def build(ctx, rebuild=False, race=False, build_include=None, build_exclude=None
         cmd = "./gradlew build"
     ctx.run(cmd)
     os.chdir(pwd)
-    ver = get_version(ctx, include_git=True, git_sha_length=7)
+    ver = get_version(ctx, include_git=True)
```

Comment:
iirc we need to specify the `git_sha_length`, otherwise some of the later scripts that upload to s3, etc., fail.

Reply:
in fact, the more I think about it, we should probably just pick a standard and use it everywhere to remove confusion.

Reply:
I made 7 the default value so it shouldn't be necessary to specify it anymore. All the scripts seems to work but I can readd it if you want